server.js

```diff
@@ -177,16 +172,15 @@ async function work(body) {
     return;
   }
 
+  // generate message using repoConfig.message
+  var message = messageGenerator (repoConfig.message,
```

Comment:
If you need to break, please put one argument on each line. No space before (
M